calendar-adjustment/metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Calendar adjustment for monthly time series",
+  "description": "Given a dataset containing monthly time series data with datestamps, normalizes the time series values by the number of days per month (with leap year detection). Suitable for monthly aggregated data (e.g. total liters per month).",
```

Comment:
thanks for asking but is that easy as to visit the current UI, put the description you want in the script description (More Info > Info > Description), save it and check how it results in the UI. Even one can check how it results if enhanced with some markdown sugar (I recommend to copy the description form the API JSON response once saved with markdown to keep the new lines and all this kind of special things).
![image](https://user-images.githubusercontent.com/743922/28816154-0a064448-76a4-11e7-8cb7-fb6ce9a6be02.png)